src/templates/gift_card.liquid

```diff
@@ -43,21 +43,21 @@
   {% endif %}
 
   {%- assign code_size = gift_card.code | format_code | size -%}
-  <span id="GiftCardDigits" class="giftcard__code">{{ gift_card.code | format_code }}</span>
+  <span id="GiftCardDigits" class="giftcard__code" data-gift-card-digits>{{ gift_card.code | format_code }}</span>
 
   <p>{{ 'gift_cards.issued.redeem' | t }}</p>
 
   <a href="{{ shop.url }}" class="button" target="_blank">{{ 'gift_cards.issued.shop_link' | t }}</a>
 
-  <div id="QrCode" class="giftcard-qr" data-identifier="{{ gift_card.qr_identifier }}"></div>
+  <canvas id="QrCode" class="giftcard-qr" data-identifier="{{ gift_card.qr_identifier }}" data-gift-card-qr></canvas>
 
   {% if gift_card.pass_url %}
     <a href="{{ gift_card.pass_url }}" class="apple-wallet">
       <img class="apple-wallet-image" src="{{ 'gift-card/add-to-apple-wallet.svg' | shopify_asset_url }}" width="120" height="40" alt="{{ 'gift_cards.issued.add_to_apple_wallet' | t }}">
     </a>
   {% endif %}
 
-  <a href="#" id="PrintGiftCard" class="print-giftcard">
+  <a href="#" id="PrintGiftCard" class="print-giftcard" data-gift-card-print>
```

Comment:
While we're here, can you switch this to use a `<button>` instead ? (You might have to reset some default CSS).

For accessibility reason, anchors should be used for anything related to content and `<button>`s for any "actions" (toggle this, expand that, print, etc).

Reply:
We didn't style the links at all, so is it worthwhile adding CSS to remove button styling instead of leaving it as is.